src/neural/CL/OpenCLTuner.cc

```diff
@@ -30,7 +30,7 @@
 #include "OpenCLParams.h"
 #include "OpenCLTuner.h"
 
-#include "utils/blas.h"
+#include "../BLAS/blas.h"
```

Comment:
Wouldn't this be better as `neural/BLAS/blas.h` ?

Reply:
I don't know. There is 1 pro 1 cons for both possible methods.

Reply:
I'm not sure if it's better, but style guide https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes indeed says to have full include paths from root.

Reply:
Pushed #include "neural/BLAS/blas.h"